java/src/jmri/jmrit/logixng/actions/configurexml/ActionListenOnBeansXml.java

```diff
@@ -34,76 +34,82 @@ public Element store(Object o) {
         Element element = new Element("ActionListenOnBeans");
         element.setAttribute("class", this.getClass().getName());
         element.addContent(new Element("systemName").addContent(p.getSystemName()));
-        
+
         storeCommon(p, element);
-        
+
         Element parameters = new Element("References");
         for (NamedBeanReference ref : p.getReferences()) {
             Element elementParameter = new Element("Reference");
-            elementParameter.addContent(new Element("name").addContent(ref.getName()));
+            if (ref.getType().name().equals(("EntryExit"))) {
+                // Use the system name when storing an EntryExit object.
+                var nxName = ref.getHandle().getBean().getSystemName();
+                elementParameter.addContent(new Element("name").addContent(nxName));
+            } else {
```

Comment:
I think it's this change that causes Windows CI to fail, that you change between system name and user name.

I'm trying to find why but haven't been successful yet. It's difficult to track down since the failing tests succeeds when only that test is run, but fails when all the LogixNG tests run.